cmd/flowrunner/testdata/flows/brochure_test.json

```diff
@@ -24,25 +23,27 @@
                 {
                     "action_uuid": "9d9290a7-3713-4c22-8821-4af0a64c0821",
                     "event": {
-                        "contacts": [
-                            {
-                                "name": "Ben Haggerty",
-                                "uuid": "ba96bf7f-bc2a-4873-a7c7-254d1927c4e3"
-                            }
-                        ],
                         "created_on": "2000-01-01T00:00:00.000000000-00:00",
-                        "text": "Hi! What is your name?",
-                        "type": "broadcast_created"
+                        "msg": {
+                            "channel": {
+                                "name": "Android Channel",
+                                "uuid": "57f1078f-88aa-46f4-a59a-948a5739c03d"
+                            },
+                            "text": "Hi! What is your name?",
+                            "urn": "tel:+12065551212",
+                            "uuid": "6bbfa705-894a-4651-8b56-5e92bbfe0e3f"
+                        },
+                        "type": "msg_created"
                     },
-                    "step_uuid": "347b55be-7be1-4e68-aaa3-04d3fbce5f9a"
+                    "step_uuid": "8707af30-d50f-440b-9803-f4a851d20f2b"
                 },
                 {
                     "event": {
                         "created_on": "2000-01-01T00:00:00.000000000-00:00",
                         "timeout_on": "2000-01-01T00:00:00.000000000-00:00",
                         "type": "msg_wait"
                     },
-                    "step_uuid": "6bce5330-ed5e-418c-a13f-9238de2a0b76"
+                    "step_uuid": "5fa51f39-76ea-421c-a71b-fe4af29b871a"
```

Comment:
Would our thrash of UUIDs be diminished on changes if we reset the seed on every test run?